### DISCUSSION TITLE
Backup

### DISCUSSION THREAD

**Discussion by bronsn** [OrcaSlicer/OrcaSlicer#2786] [General]
Hey,

is there no pissibilioty to make a backup from all Settings?
Actually i only found an option to save the filament settings.

**Comment by BillyGriffiths** [comment]
**I will second this.** I need to be able to back up / export all my printer/filament/process user presets as a lot of time goes into them. 

Failing that, can anyone tell me where on my computer (mac) all my customised files are saved? I cannot find the "export" option under File->Export

<img width="527" alt="image" src="https://github.com/user-attachments/assets/ba0146c1-2aba-44d9-8635-a937d36de688">
